docker/docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+---
+version: "3"
+services:
+  storywise:
+    image: mihainueleanu/storywise:latest
+    container_name: storywise
+    hostname: storywise
+    environment:
+      - PUID=1000
+      - PGID=1000
```

Comment:
Interesting. I only use docker for running local workloads (for development purposes, and for hosting things I use kubernetes), and this is the first time I'm seeing PUID, PGID being set in a docker-compose. Do you happen to have some article as to why this is good practice in docker? I'm just curious to know